Gemfile

```diff
@@ -20,7 +20,7 @@ gem 'paper_trail' # Save histories of record changes related to surveys
 gem "kt-paperclip" # used by Course and UserProfile for file attachments.
 gem 'sidekiq' # Framework for running background worker jobs
 gem 'sidekiq-unique-jobs' # Plugin to prevent duplicate jobs in the sidekiq queue
-gem 'sidekiq-cron' # Plugin for cron-style recurring jobs in Sidekiq
+gem 'sidekiq-cron', '~> 1.12.0' # Plugin for cron-style recurring jobs in Sidekiq
```

Comment:
This is not necessary, as the lock file will ensure that the new version gets used. Typically we include as few explicit version constraints in the Gemfile as possible.

Reply:
The updated Gemfile.lock will ensure that 1.12.0 is used instead. To update the lock file, you need to start by either making a change like this to the Gemfile, or removing the entries for that gem from the lock file and then re-installing the gems to regenerate the lock file, or using bundler from the command line to update a particular gem. But if you use the Gemfile method like you did, it's fine to then remove the version constraint after you update the lock file.